### DISCUSSION TITLE
Unknown TypeError in console

### DISCUSSION THREAD

**Discussion by prajjwalarora** [nuxt/nuxt#16502] [Questions] [answered]
Hi, 
I've encountered an error in the console from companion-bubble.js while experimenting with rc-12 and typescript.
At first, I thought it was because of my code. but I scaffolded a fresh rc-12 project and it's there in the fresh project as well.

So my question is, is there any way we can fix this error or this is a bug in this release?
![Screenshot 2022-10-31 at 3 45 29 PM](https://user-images.githubusercontent.com/57281080/198985193-75dbb062-8287-43b0-8b36-7014b7b2a272.png)

**Comment by sewalsh** [comment, marked as answer]
This error appears to be a side effect from using a Chrome extension called 'Loom'. https://stackoverflow.com/a/72091058/965333

Disable it and see if the error disappears.

**Reply by prajjwalarora** [reply]
Thanks a lot. this worked.
